STM32Cube/Core/Src/main.c

```diff
@@ -136,6 +136,42 @@ Otits_Result_t test_defaultTaskFail() {
 	res.outcome = TEST_OUTCOME_FAILED;
 	return res;
 }
+
+Otits_Result_t test_taskStackUsage() {
+    Otits_Result_t res;
+    const int numTasks = 9;
+    int stackWatermarks[numTasks];
+
+    stackWatermarks[0] = uxTaskGetStackHighWaterMark(logTaskhandle);
+    stackWatermarks[1] = uxTaskGetStackHighWaterMark(VNTaskHandle);
+    stackWatermarks[2] = uxTaskGetStackHighWaterMark(trajectoryTaskHandle);
+    stackWatermarks[3] = uxTaskGetStackHighWaterMark(stateEstTaskHandle);
+    stackWatermarks[4] = uxTaskGetStackHighWaterMark(canhandlerhandle);
+    stackWatermarks[5] = uxTaskGetStackHighWaterMark(healthChecksTaskHandle);
+    stackWatermarks[6] = uxTaskGetStackHighWaterMark(controllerHandle);
+    stackWatermarks[7] = uxTaskGetStackHighWaterMark(flightPhaseHandle);
+    stackWatermarks[8] = uxTaskGetStackHighWaterMark(oTITSHandle);
+
```

Comment:
someone should run OTITS to check the results of this so we dont stack overflow by accident

Reply:
We should give OTITs more space, its currently chilling at 49 bytes remaining on its stack during runtime.